src/controlers/expences.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const expenceServices = require('../services/expences');
+const userServices = require('../services/users');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
```

Comment:
Consider validating input parameters like 'userId', 'categories', 'from', and 'to' before passing them to service functions.

Reply:
`if (!userId || !categories || !from || !to) {
    res.sendStatus(400);

    return;
  }`

  this validation causes 5 tests to fail, this may be due to the logic of the test.js file